helmfile/charts/elasticsearch-slm/scripts/slm-retention.bash

```diff
@@ -75,12 +76,14 @@ function remove_old_snapshots {
     local idx=0
     local snapshots
     local snapshot_count
+    local snapshots_to_delete=""
```

Comment:
Could you add a log line here also to make it more obvious what's going on?
Something like
```bash
echo "Checking for old snapshots."
```